### ISSUE TITLE
Non-Latin Characters Garbled

### ISSUE THREAD

**Issue by cawoodm** [rannn505/node-powershell#132]
It seems characters like `ä` or `ü` are incorrectly encoded in the resulting javascript string.

### PowerShell
```powershell
echo "bär"

bär
```

### Node-PowerShell
```
let PS = require('node-powershell');
const ps = new PS({
  executionPolicy: 'Bypass',
  noProfile: true
});
ps.addCommand('echo bär');
ps.invoke()
.then(output => {
  console.log(output);
})

b├ñr
```
As you can see the buffer is mangled to `b├ñr`:
```
b: 98 OK
ä: 195 164
r: 114 OK
```

![image](https://user-images.githubusercontent.com/498834/107869987-b96bd880-6e94-11eb-8187-d22de1a2ee77.png)

**Comment by cawoodm** [comment]
Apparently PowerShell is receiving `ä` encoded as 5 bytes `226` `148` `156` `195` and `177`!

```
ps.addCommand('$str="bär";echo $str; echo $([System.Text.Encoding]::UTF8.GetBytes($str))');
...
b├ñr
98  
226 
148 
156 
195
177
114
```

**Comment by thomas3577** [comment]
@cawoodm Hi, do you have a solution or workaround for this problem yet? In the latest version, the problem is still present.

**Comment by cawoodm** [comment]
Nope, I gave up on this library 🤷‍♂️ and went for `node-powershell`.

https://github.com/cawoodm/powowshell/blob/master/ide/package.json

**Comment by thomas3577** [comment]
@cawoodm I am using node-powershell v5.0.1. And I am not familiar with using profiles for PowerShell. But now I was able to solve my problem by removing noProfile from PS-Config. But I'm not sure that's a good idea.

Anyway, "Bär" is now displayed as a "Bär" and no longer as "B├ñr".  :-)
